### DISCUSSION TITLE
Sell-Chest Whitelist and Sell-Timer

### DISCUSSION THREAD

**Discussion by XPlliarmus** [BG-Software-LLC/WildChests#173] [Pending Feature Requests]
It would be great if you could set the interval for sell-chests to sell. This way there could be different tiers of sell chests that sell faster than others.

With that said it would also be great to have a whitelist for Sell-Chests. This way you could give out sell chests for a specific item/farm to give sell-chests for variability and not just have one sell-chest that does everything.

Both features are highly welcomed to be added :)

**Comment by InvisibleSpiders** [comment]
I know this is about 2 years old now, but +1 for Sell Timer intervals!
